tests/integration_test.rs

```diff
@@ -1,11 +1,12 @@
-use std::collections::HashSet;
-use std::fs;
 use country_boundaries::{self, BoundingBox, CountryBoundaries, LatLon};
+use std::collections::HashSet;
+use std::fs::File;
+use std::io::BufReader;
 
 #[test]
 fn return_correct_results_at_cell_edges() {
-    let buf = fs::read("./data/boundaries360x180.ser");
-    let boundaries = CountryBoundaries::from_reader(buf.unwrap().as_slice()).unwrap();
+    let buf = BufReader::new(File::open("./data/boundaries360x180.ser").unwrap());
+    let boundaries = CountryBoundaries::from_reader(buf).unwrap();
```

Comment:
I had it like this at the beginning, but changed it to this because it is
- shorter (=easier to read) and
- it is likely faster because the buffer size is just made to be the same size of the file, i.e. the buffer does not need to be re-allocated in (by default) 8KB steps

I posted in the Rust forum about it:
https://users.rust-lang.org/t/performance-reading-file-parse-from-io-read-vs-from-u8/91948

Reply:
thx, makes sense, although on the second thought I would just use the `include_bytes!` macro and not deal with the real time reading at all - easier in tests